stable/pipelines/templates/pipelines-statefulset.yaml

```diff
@@ -198,6 +198,8 @@ spec:
           {{- end }}
             sleep 20;
             ./pipelines-k8s;
+            echo "Changing the owner of the system.yaml to pipelines user"
+            chown 1066:1066 /opt/jfrog/pipelines/var/etc/system.yaml
```

Comment:
We should have the id configurable. Also need to think about openshift with its id ranged. cc @peters95